velociraptor/catalogue/translator.py

```diff
@@ -1526,6 +1486,7 @@ def VR_to_SOAP(particle_property_name: str) -> str:
             "exclusivesphere.100kpc.linearmassweightedironfromsniaoverhydrogenofstars",
             -1,
         ),
+        "searchradius.search_radius": ("boundsubhalo.encloseradius", -1),
```

Comment:
I see that the `search radius` field was not in the translator file before.  Do we now need this field somewhere?

Reply:
It is used by the morphology pipeline when we are deciding what region of the simulation to read in.

https://github.com/SWIFTSIM/morpholopy/blob/f4991a475c74ab60f436f76f31764d0f35a7c5f8/morpholopy/galaxy_data.py#L448